### PR TITLE
Documentation - Add docs for Visual Editor updates

### PR DESCRIPTION
**Pull request by bttf**
New topics added to Visual Editor page:
- Drag and drop feature
- SDK warnings and explanations
- Debug panel

### CODE REVIEW

File: docs/docs/visual-editor.mdx [hunk 1]
```diff
@@ -113,6 +113,50 @@ When you're using the Element Selector, after you pick an element to edit, you'l
...
+
+#### SDK is not present
+
+This message is indicating that the page you are editing does not have a GrowthBook SDK loaded. An SDK is required to be loaded and running on your page for your experiments to run.
+
+<img style={{ width: '400px'}} src="/images/visual-editor-sdk-not-detected.png" alt="SDK is not present"/>
+
+#### SDK is not up to date
+
+This message is indicating that your GrowthBook SDK is working but not up todate. You may encounter unexpected behavior as a result.
```
Comment (august-growthbook) [inline docs/docs/visual-editor.mdx:144]: Minor typo here, "up todate" --> "up to date".

File: docs/docs/visual-editor.mdx [hunk 2]
```diff
@@ -113,6 +113,50 @@ When you're using the Element Selector, after you pick an element to edit, you'l
...
+
+#### Hash attribute is missing
+
+This message is indicating that the hash attribute defined for your visual experiment is not being provided to the SDK. The hash attribute is required for the SDK to bucket users and without it, it cannot run the visual experiment.
+
+<img style={{ width: '400px'}} src="/images/visual-editor-hash-attr-missing.png" alt="Hash attribute is missing"/>
+
+### Debug Panel
+
+The Visual Editor now comes with a Debug Panel to help diagnose any issues with your SDK configuration and URL Targeting rules. Use this to help diagnose your own errors, or provide a screenshot to support when reaching out to help provide context.
```
Comment (august-growthbook) [inline docs/docs/visual-editor.mdx:156]: "to support" --> "to our Support team"

**Review by august-growthbook — APPROVED** [review]
2 minor typos, approved, looks great! Glad to see the screenshots, and the instructions are very clear.

File: docs/docs/visual-editor.mdx (hunk 1 above)
Comment (bttf) [inline docs/docs/visual-editor.mdx:144]:
```suggestion
This message is indicating that your GrowthBook SDK is working but not up to date. You may encounter unexpected behavior as a result.
```

File: docs/docs/visual-editor.mdx (hunk 2 above)
Comment (bttf) [inline docs/docs/visual-editor.mdx:156]:
```suggestion
The Visual Editor now comes with a Debug Panel to help diagnose any issues with your SDK configuration and URL Targeting rules. Use this to help diagnose your own errors, or provide a screenshot to our Support team when reaching out to help provide context.
```

**Review by august-growthbook — APPROVED** [review]

**Pull request merged**